argocd_instance/argo-cd-cr-component-properties.adoc

```diff
@@ -26,8 +26,13 @@ include::modules/proc_configuring-by-using-the-ocp-web-console.adoc[leveloffset=
 // Configuring the NotificationsConfiguration CR by using the CLI
 include::modules/proc_configuring-notificationsconfiguration-crd-by-using-the-ocp-cli.adoc[leveloffset=+2]
 
+// Enabling Config Management plugins in an Argo CD CR
+include::modules/con-enabling-config-management-plugins-argo-cd-cr.adoc[leveloffset=+1]
```

Comment:
Should this come higher in the assembly? Perhaps, after `Enabling notifications with an Argo CD instance`. It seems out of place down here.

Reply:
Fixed.